data/pid_data.py

```diff
@@ -1148,6 +1206,16 @@
        'notes': 'Controls if the fixture will treat key seven on the MCX as a lamp power-off button.',
        'value': 0x8002},
 
+      {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'enabled', 'type': 'uint16'}]},
+       'get_sub_device_range': 0,
+       'name': 'FIXTURE_ID',
+       'set_request': {'items': [{'name': 'enabled', 'type': 'uint16'}]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'notes': 'Controls if the fixture will treat key seven on the MCX as a lamp power-off button.',
```

Comment:
This notes field looks like it was copied from the PID above and doesn't match the PID name.